crates/nu-table/src/table.rs

```diff
@@ -147,18 +148,34 @@ fn draw_table(
     }
 
     let with_header = table.with_header;
-    let with_footer = with_header && need_footer(config, (&table.data).size().0 as u64);
+    let use_domterm_html = config.use_domterm_features;
+    let with_footer = ! use_domterm_html && with_header && need_footer(config, (&table.data).size().0 as u64);
     let with_index = table.with_index;
 
     if with_footer {
         table.data.duplicate_row(0);
     }
 
-    let mut table = Builder::custom(table.data).build();
+    let mut b = Builder::custom(table.data);
+    if with_header {
+        b.with_header();
+    };
+
+    let mut table = b.build();
     load_theme(&mut table, color_hm, theme, with_footer, with_header);
     align_table(&mut table, alignments, with_index, with_header, with_footer);
     table_trim_columns(&mut table, termwidth, &config.trim_strategy);
 
+    if use_domterm_html {
+        //FUTURE - currently causes misaslignment with floating header
```

Comment:
Could you provide a screenshot what the issue is?

Reply:
The misalignment isn't a problem with `tabled` but with how "floating table headers" (see [this](https://github.com/nushell/nushell/pull/6786#issuecomment-1281369803)) is implemented. To implement floating headers, there is JavaScript that temporarily clones the table (triggered by an "intersection observer): one for the floating header and one for the body. The first is positioned at the top of the window, and has the table body invisible; the other scrolls normally.

If the table padding is removed, these two tables may not align properly, which ruins the effect. Removing the extra `<p>` elements created by `table_to_html` seems to help. Some JavaScript (such as explicitly setting the table width of the clone) may help.

This is something I have to figure out.